include/graphblas/bsp1d/io.hpp

```diff
@@ -823,6 +823,49 @@ namespace grb {
 		return ret;
 	}
 
+	template<
```

Comment:
The backend should define performance semantics (see other primitives for examples, should be straightforward in this case)